services/datamanager/builtin/config.go

```diff
@@ -0,0 +1,102 @@
+package builtin
+
+import (
+	"go.viam.com/rdk/components/sensor"
+	"go.viam.com/rdk/internal/cloud"
+	"go.viam.com/rdk/services/datamanager/builtin/capture"
+	datasync "go.viam.com/rdk/services/datamanager/builtin/sync"
+)
+
+// Sync defaults.
+const (
+	// Default time to wait in milliseconds to check if a file has been modified.
+	defaultFileLastModifiedMillis = 10000.0
+	// defaultMaxParallelSyncRoutines is the maximum number of sync goroutines that can be running at once.
+	defaultMaxParallelSyncRoutines = 1000
+	// defaultDeleteEveryNth temporarily public for tests.
```

Comment:
Can we add a comment here for what deleteEveryNth controls, similar to the above.

Reply:
done